packages/meteor-accounts-saml/server/saml_utils.js

```diff
@@ -8,11 +6,10 @@ import xmldom from 'xmldom';
 import querystring from 'querystring';
 import xmlbuilder from 'xmlbuilder';
 import array2string from 'arraybuffer-to-string';
-
 // var prefixMatch = new RegExp(/(?!xmlns)^.*:/);
 
 
-SAML = function(options) {
+const SAML = function(options) {
```

Comment:
wouldn't be better this?
```suggestion
export const SAML = function(options) {
```